cluster-endpoints/src/endpoint_stremers.rs

```diff
@@ -1,9 +1,13 @@
-use solana_lite_rpc_core::types::{BlockStream, ClusterInfoStream, SlotStream, VoteAccountStream};
+use solana_lite_rpc_core::{
+    structures::account_data::AccountStream,
+    types::{BlockStream, ClusterInfoStream, SlotStream, VoteAccountStream},
+};
 
 /// subscribers to broadcast channels should assume that channels are not getting closed unless the system is shutting down
 pub struct EndpointStreaming {
     pub blocks_notifier: BlockStream,
     pub slot_notifier: SlotStream,
     pub vote_account_notifier: VoteAccountStream,
     pub cluster_info_notifier: ClusterInfoStream,
+    pub processed_account_stream: Option<AccountStream>,
```

Comment:
using Optional Stream here is a new concept to state that the feature is not configured. Isn't that a bit implicit?

Reply:
If the feature is not configured then it is disabled, so optional in nature,
I think it is correct way.